include/ck_tile/core/numeric/half.hpp

```diff
@@ -129,8 +129,8 @@ constexpr double fp16_to_double_hip(const fp16_hip_t& x)
 CK_TILE_HOST_DEVICE
 constexpr fp16_hip_t float_to_fp16_hip(const float& x)
 {
-    return __float2half(x);
-    // return static_cast<fp16_hip_t>(x);
+    // return __float2half(x);
```

Comment:
@rocking5566 Thanks! It will be great if you can create an issue documenting the problem (because it seems that there is some back-and-forth about it), and mention in a comment above this line for reference. Thanks!

Reply:
https://github.com/ROCm/composable_kernel/issues/1294
I open a ticket to record this issue. 
I will also share with ck team member to avoid similar issue happen in the future.